sites/ademe/src/backend/templates/richie/base.html

```diff
@@ -5,7 +5,7 @@
 
 {% block topbar_contact %}
 <li class="topbar__item topbar__item--cta">
-    <a href="https://www.fun-services.help/hc/">{% trans "Help" %}</a>
+    <a href="/contact/">{% trans "Contact us" %}</a>
```

Comment:
Since it is a cms page, It's better to do like here https://github.com/openfun/richie-site-factory/blob/master/sites/funcampus/src/backend/templates/richie/base.html#L23 otherwise it may change language or break if the slug changes.

Reply:
nice catch, i was doing it the crappy way..

Reply:
another way to say it :stuck_out_tongue: